DM.MovieApi.IntegrationTests/MovieDb/TV/ApiTVShowRequestTests.cs

```diff
@@ -249,5 +249,57 @@ public async Task GetPopularAsync_CanPageResults()
             await ApiResponseUtil.AssertCanPageSearchResponse( "none", minimumPageCount, minimumTotalResultsCount,
                 ( _, page ) => _api.GetPopularAsync( page ), x => x.Id );
         }
+
+        [TestMethod]
+        public async Task GetTvShow_SeasonInfo_GameOfThrones_ReturnsAllValues()
+        {
+            var expFirstAirDate = new DateTime(2015, 04, 12);
+            const string expName = "Season 5";
+            const string expOverview = "The War of the Five Kings, once thought to be drawing to a close, is instead entering a new and more chaotic phase. Westeros is on the brink of collapse, and many are seizing what they can while the realm implodes, like a corpse making a feast for crows.";
+            const int expEpisodeCount = 10;
+            const int expSeasonNumber = 5;
+
+            Episode expEpisodeOne = new Episode()
```

Comment:
@mejackhumphrey  Left hand side of the expression can be `var` since the right hand side is explicit. 

Right hand side should remove the redundant / empty argument, re: `new Episode`, not `new Episode()`

```
var expEpisodeOne = new Episode
{
...
}
```